pkg/controllers/provisioning/suite_test.go

```diff
@@ -2583,6 +2584,447 @@ var _ = Describe("Provisioning", func() {
 			})
 		})
 	})
+
+	Context("AutoRelaxMinValues", func() {
```

Comment:
Note: I haven't looked deeply over these tests -- going to iron-out the core logic and then look more scrupulously over the testing